pkg/cmd/build/buildkit.go

```diff
@@ -286,7 +281,11 @@ func solveBuild(ctx context.Context, c *client.Client, opt *client.SolveOpt, pro
 			// not using shared context to not disrupt display but let it finish reporting errors
 			return progressui.DisplaySolveStatus(context.TODO(), "", nil, w, plainChannel)
 		case "deploy":
-			err := deployDisplayer(context.TODO(), plainChannel)
+			err := deployDisplayer(context.TODO(), plainChannel, &types.BuildOptions{OutputMode: "deploy"})
```

Comment:
NIT: I'd just go with a single string `deploy` instead of creating the buildOptions but It's ok like this too